src/components/heading/heading.component.js

```diff
@@ -29,7 +30,7 @@ class Heading extends React.Component {
     /**
      * Defines the title for the heading.
      */
-    title: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+    title: PropTypes.node,
```

Comment:
shouldn't it be just a string?

Reply:
After @edleeks87 noticed, mistake with backLink i reverted all types as they were before.